src/pages/dbot/_dbot-video.tsx

```diff
@@ -62,7 +62,12 @@ const VideoWrapper = styled.div`
     max-width: 79.2rem;
     width: 100%;
 `
-const StepCommon = css`
+
+type StepCommonTypes = {
+    no_margin: string
+}
```

Comment:
```suggestion
type StepCommonType = {
    no_margin: string
}
```
It should be `Type` instead of pluralising it.